components/Header/Header.js

```diff
@@ -145,7 +192,8 @@ class Header extends React.Component {
 
 Header.contextTypes = {
     executeAction: PropTypes.func.isRequired,
-    getUser: PropTypes.func
+    getUser: PropTypes.func,
+    intl: React.PropTypes.object.isRequired
```

Comment:
If you revert you don't need to import this context too - I guess less overhead.